From: John Zupin <johnz@lunarg.com>
Date: Thu, 3 Feb 2022 14:39:24 -0700
Subject: patch: update add_copyright.py"

---
 utils/add_copyright.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/add_copyright.py b/utils/add_copyright.py
index 7ea0897..23b9f46 100755
--- a/utils/add_copyright.py
+++ b/utils/add_copyright.py
@@ -62,7 +62,7 @@ def filtered_descendants(glob):
     """Returns glob-matching filenames under the current directory, but skips
     some irrelevant paths."""
     return find('.', glob, ['third_party', 'external', 'build*', 'out*',
-                            'CompilerIdCXX', '.venv', 'glslang'])
+                            'CompilerIdCXX', '.venv', 'glslang', 'obj-x86_64-linux-gnu'])
 
 
 def skip(line):
